proposals/0085-additional-fee-collector-constraints.md

```diff
@@ -0,0 +1,61 @@
+---
+simd: "0085"
+title: Additional Fee Collector Constraints
+authors:
+  - Justin Starry
+category: Standard
+type: Core
+status: Draft
+created: 2023-11-05
+feature: https://github.com/solana-labs/solana/issues/33888
+---
+
+## Summary
+
+Burns fees if they would otherwise be collected into an account that violates
+one of the following constraints:
+
+1. Must be a system owned account
+2. Must be rent-exempt
+
+This applies to both transaction fees and rent fees. Note that the "rent-exempt"
+constraint was already added for rent fee collection in
+[Feature 30151](https://github.com/solana-labs/solana/issues/30151).
+
+## Motivation
+
+1. Since fee collection occurs outside of the runtime, it's generally a good
+   idea to reduce the number of account modification edge cases.
+2. Prevent new rent paying accounts from being created
+
+## Alternatives Considered
+
+NA
+
+## New Terminology
+
+NA
+
+## Detailed Design
+
+At the end of a block, validators MUST NOT distribute fees to accounts that are
+not system owned and/or rent-exempt. Instead, they MUST burn the fees by not
+distributing them to anyone.
```

Comment:
```suggestion
At the end of a block, validators MUST NOT distribute fees to accounts that are
rent-paying or owned by a non-system program. Instead, they MUST burn the fees by not
distributing them to anyone.
```
Nit. I thought this was a little easier to follow without one set of negatives.

Reply:
I flipped this to a set of positives which I hope is even clearer, thanks for pointing this out